content/writing-packages.md

```diff
@@ -47,6 +49,8 @@ The `package.js` file is the main file in every Meteor package. This is a JavaSc
 
 In this guide article, we will go over some important points for building packages, but we won't explain every part of the `package.js` API. To learn about all of the options, [read about the `package.js` API in the Meteor docs.](http://docs.meteor.com/#/full/packagejs)
 
+Don't forget to run `meteor add [my-package]` once you have finished developing your package in order to use it, even if the package is for internal use only and you don't intend to publish the package to Atmosphere.
```

Comment:
Not 100% sure what you want clarified, but I tweaked the text. Let me know if this is what you wanted.